code/crates/engine/src/util/streaming.rs

```diff
@@ -39,6 +39,7 @@ pub enum StreamContent<T> {
     Data(T),
 
     /// Fin must be set to true.
+    /// Q: Why not just leave out the bool then?
```

Comment:
This was done to match the original Starknet protos, but those have since removed the boolean, meaning we can do so as well.